CourseApp/ClassTaskFolder/ClassTask.cs

```diff
@@ -0,0 +1,20 @@
+namespace CourseApp.Class
+{
+    using System;
+    using System.Collections.Generic;
+
+    public class ClassTask
+    {
+        public ClassTask(string name, string extension, double weight, string weightModificator)
+        {
+            Document file = new FileCreator(name, extension, weight, weightModificator);
+            Console.WriteLine($"Your file: {file.Display()}");
```

Comment:
It's not good to print inside constructor

Reply:
Single Responcibility